apps/prairielearn/src/pages/home/home.ejs

```diff
@@ -487,19 +487,12 @@
       <% } %> <!-- isAuthenticated -->
     </main>
 
-    <footer class="footer font-weight-light text-light <%= isAuthenticated ? 'tiny text-center' : 'small text-right bg-dark py-3' %>">
-      <% if (isAuthenticated) { %>
-      <div class="bg-flourish h-flourish"></div>
-      <% } %>
-      <div class="<%= isAuthenticated ? 'bg-secondary p-1' : 'container' %>">
-        <% if (config.homepageFooterText) { %>
+    <% if (config.homepageFooterText && config.homepageFooterTextHref) { %>
```

Comment:
This means the footer is now only shown if `homepageFooterText` and `homepageFooterTextHref` are defined, which in turn is only true in China at the moment.